.github/workflows/callable-scan.yaml

```diff
@@ -128,7 +133,7 @@ jobs:
       - name: Get Images
         id: get-images
         run: |
-          IMAGES=$(uds zarf tools yq -r -o=json -I=0 '[.components.[].images] | flatten | unique | map(select((test(".sig") | not) and (test(".att") | not)))' ${{ inputs.zarfYamlPath }})
+          IMAGES=$(uds zarf tools yq -r -o=json -I=0 '[.components.[].images? // []] | flatten | unique | map(select((test(".sig") | not) and (test(".att") | not)))' ${{ inputs.zarfYamlPath }})
```

Comment:
I ran into *exactly* this problem in the Strimzi package last week - I like this solution, it was better than mine.

Reply:
Thanks! Trying to resolve all the issues I've been seeing across repos